src/Interface/Modules/Render/Screenshot.cc

```diff
@@ -88,11 +97,25 @@ void Screenshot::saveScreenshot(const QString& filename)
   screenshot_.save(filename);
 }
 
+void Screenshot::saveScreenshotFromPath()
+{
+  index_++;
+  auto fileName = screenshotFileFromPreferences();
+  if (!fileName.isEmpty())
+  {
+    QMessageBox::information(nullptr, "ViewScene Screenshot", "Saving ViewScene screenshot to: " + fileName);
+    screenshot_.save(fileName);
+  }
+}
+
+QString Screenshot::screenshotFileFromPreferences() const
+{
+  return screenshotDirectory() + QString("/viewScene_%1_%2.png").arg(QDateTime::currentDateTime().toString("yyyy.MM.dd.HHmmss.zzz")).arg(index_);
```

Comment:
Is there an easy way to add the viewscene number? it looks like it tracks the index for each viewscene window.   How about the network name?  These could be pushed until after the release.